spades.wdl

```diff
@@ -0,0 +1,79 @@
+task spades {
+    String outputDir
+    String? preCommand
+    File? read1
+    File? read2
+    File? singleRead
```

Comment:
Shouldn't one of the inputs be mandatory?

Reply:
Unfortunately, this is not possible. Unless you want to do some crazy logic magic in bash. (And I do not want to do that)